### PR TITLE
add a diagram to introduce workflow of casting video player

### PR DESCRIPTION
**Pull request by nicelyjust**
add a diagram, in order to illustrate more specifically steps

### CODE REVIEW

**Review by andy31415 — APPROVED** [review]

**Review by chrisdecenzo — APPROVED** [review]

**Review by yunhanw-google — APPROVED** [review]

File: examples/tv-casting-app/APIs.md [hunk 1]
```diff
@@ -67,6 +67,9 @@ Next, you're ready to:
 1. [Read](#read-operations) endpoint attributes like playback state.
 1. [Subscribe](#subscriptions) to playback events.
 
+In order to illustrate more specifically these steps,refer to the below figure
+![workflow of casting video player](./diagram/workflow_of_casting_video_player.png)
```
Comment (sharadb-amazon) [inline examples/tv-casting-app/APIs.md:71]:
```suggestion
In order to illustrate these steps, refer to the figure below 
![workflow of casting video player](./diagram/workflow_of_casting_video_player.png)
```

**Review by sharadb-amazon — COMMENTED** [review]
1.  Why does the "discover" arrow from the CastingClient just loop back upon itself? Shouldn't it touch the CastingPlayer or something else on the network?
2.  Note that the CastingClient does not always need to send the User Directed Commissioning request. After it has been commissioned by the CastingPlayer once, it is simply able to re-establish CASE the next time it wants to cast to the same CastingPlayer (using a cached nodeId, fabricIndex). This is taken care of under-the-hood, by the VerifyOrEstablishConnection API.

File: examples/tv-casting-app/APIs.md (hunk 1 above)
Comment (nicelyjust) [reply examples/tv-casting-app/APIs.md]: Thanks for your suggestion.

Comment (nicelyjust) [reply examples/tv-casting-app/APIs.md]: Pls check it again @sharadb-amazon

**Pull request merged**